src/LeagueToolkit/IO/ReleaseManifestFile/ReleaseManifestChunkingParameter.fbs

```diff
@@ -0,0 +1,10 @@
+﻿namespace LeagueToolkit.IO.ReleaseManifestFile;
+
+table ReleaseManifestChunkingParameter
+{
+    ID: ushort;
+    Unknown1: byte;
+    Unknown2: byte;
+    Unknown3: byte;
```

Comment:
`Unknown3` is specified as an `int` in the current specification, are you sure this is correct ?

Reply:
Fixed.